### DISCUSSION TITLE
SFRKC30.AT2 (QFN32) not showing in ISP

### DISCUSSION THREAD

**Discussion by Alessandro-Mariotti** [keirf/flashfloppy#756] [Q&A]
Hello everybody,
I bought thi smodel of gotek, the micro is a AT32F415, the 4mm square version, I can't install the firmware in any way. 
Connected to Windows, without jumper, it is not recognized, while with the jumper installed it is recognized by Windows (as DFU in FS mode), but there is no way to make it appear in ISP.
I have installed and reinstalled several times the driver that is downloaded together with the ISP, I have tried with version 2.0.06, 1.5.46 and multi-port version 2.0.07, but none of them show the connected device.
Any Idea?

**Comment by keirf** [comment]
Not an issue I've heard of before I'm afraid.
